package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "powerbi-client",
-  "version": "2.14.1",
+  "version": "2.16.5",
   "description": "JavaScript library for embedding Power BI into your apps. Provides service which makes it easy to embed different types of components and an object model which allows easy interaction with these components such as changing pages, applying filters, and responding to data selection.",
   "main": "dist/powerbi.js",
-  "typings": "dist/powerbi-client.d.ts",
+  "types": "dist/powerbi-client.d.ts",
```

Comment:
package.json was not merged correctly.
The script section is outdated and still contains typings install which is what is causing the travis build to fail

Reply:
Worked!